core/src/main/kotlin/entity/Guild.kt

```diff
@@ -71,6 +71,11 @@ class Guild(
      */
     val approximatePresenceCount: Int? get() = data.approximatePresenceCount.value
 
+    /**
+     * The welcome screen of a Community guild, shown to new members.
+     */
+    val welcomeScreen: WelcomeScreen? get() = data.welcomeScreen.value
```

Comment:
Should we really be returning a JSON object here? It's not how we usually do this

Reply:
right I'll have to add a core representation of this now that it also has some sort of behavior